pkg/ebpf/c/vmlinux.h

```diff
@@ -789,6 +789,21 @@ struct ipv6hdr {
 struct tcphdr {
 	__be16 source;
 	__be16 dest;
+	__be32 seq;
+	__be32 ack_seq;
+	__u16 res1:4;
+	__u16 doff:4;
+	__u16 fin:1;
+	__u16 syn:1;
+	__u16 rst:1;
+	__u16 psh:1;
+	__u16 ack:1;
+	__u16 urg:1;
+	__u16 ece:1;
+	__u16 cwr:1;
+	__be16 window;
+	__sum16 check;
+	__be16 urg_ptr;
```

Comment:
Do we really need all those fields here?
Won't adding `doff` be enough?

Reply:
adding `doff` isn't enough by itself. we have to add akk the bitfields of this `__u16`. other fields of the struct can be removed.